go.mod

```diff
@@ -1,21 +1,21 @@
 module github.com/prysmaticlabs/prysm/v3
 
-go 1.19
+go 1.18
```

Comment:
Any reason to downgrade?

Reply:
apologies no, this is just for the machine I was testing this on (need to update the go binary!)  I will revert this once I can also remote the replace directive.

Reply:
fixed